.tekton/integration-tests/tasks/rhtap-install.yaml

```diff
@@ -35,5 +42,46 @@ spec:
         git clone "$(params.git-url)" .
         git checkout "$(params.git-revision)"
 
+        # Export the environment variables from the secret
+        cat /workspace/config.env > config.sh
+        chmod +x config.sh
+        source ./config.sh
+
+        # Path to your values.yaml.tpl file
+        tpl_file="rhtap-cli/charts/values.yaml.tpl"
+
+        # Create the new integrations section
+        new_integrations=$(cat <<EOF
+        integrations:
+          github:
+            id: "${GITHUB__APP__ID}"
+            clientId: "${GITHUB__APP__CLIENT__ID}"
+            clientSecret: "${GITHUB__APP__CLIENT__SECRET}"
+            publicKey: |
+        $(echo "${GITHUB__APP__PRIVATE_KEY}" | sed 's/^/      /')
+            token: "${GITOPS__GIT_TOKEN}"
+            webhookSecret: "${GITHUB__APP__WEBHOOK__SECRET}"
+          gitlab:
+            token: "${GITLAB__TOKEN}"
+          quay:
+            dockerconfigjson: |
+        $(echo "${QUAY__DOCKERCONFIGJSON}" | sed 's/^/      /')
+            token: "${QUAY__API_TOKEN}"
+        EOF
+        )
+
+        # Use awk to replace the integrations section
+        awk -v new_integrations="$new_integrations" '
+          BEGIN { found = 0 }
+          /^# integrations:/ { found = 1; print new_integrations; next }
+          found && /^# rhtap-dh/ { found = 0 }
+          !found { print }
+        ' "$tpl_file" > tmpfile && mv tmpfile "$tpl_file"
+
+        echo "make build"
         make build
-        ./bin/rhtap-cli deploy --config ./config.yaml --kube-config "$KUBECONFIG" --debug
+
+        echo "install"
+        ./bin/rhtap-cli deploy --config ./config.yaml --kube-config "$KUBECONFIG" --debug --log-level=debug
```

Comment:
I don't believe we should be using --debug --log-level=debug since we are attempting to add secrets to the config. This will essentially just print our secrets in plain text in the logs.

Reply:
i see, I am debugging this CI by using it

Reply:
Done